File: app/src/main/java/ru/mail/sporttogether/auth/core/listeners/OnLoginCompleteListener.java
package ru.mail.sporttogether.auth.core.listeners;


import ru.mail.sporttogether.auth.core.SocialNetworkError;

/**
 * Created by said on 13.11.16.
 */

public interface OnLoginCompleteListener {
    void onSuccess();

    void onCancel();

    void onError(SocialNetworkError socialNetworkError);
}

File: app/src/main/java/ru/mail/sporttogether/auth/core/listeners/OnLoginCompleteListener.kt
package ru.mail.sporttogether.auth.core.listeners


import ru.mail.sporttogether.auth.core.SocialNetworkError

/**
 * Created by said on 13.11.16.
 */

interface OnLoginCompleteListener {
    fun onSuccess(ID: Int)

    fun onCancel()

    fun onError(socialNetworkError: SocialNetworkError)
}
